### PR TITLE
Add power bladesworn 

### PR DESCRIPTION
**Pull request by MuwuM**
Also hide editor when author == editor (as it looked strange)

### CODE REVIEW

File: docs/_includes/build-sidebar.html [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {% if page.author_profile or layout.author_profile or page.sidebar %}
   <div class="sidebar sticky">
   {% if page.author_profile or layout.author_profile %}{% include author-profile.html %}{% endif %}
-  {% if page.editor_profile or layout.editor_proflie %}
-    <hr/>
-    {% include editor-profile.html %}
+  {% if page.editor_profile or layout.editor_profile %}
+  {% if page.author == page.editor %}{% else %}
```
Comment (berdandy) [inline docs/_includes/build-sidebar.html]: Prefer `{% if page.author != page.editor %}`

File: docs/_layouts/build-sidebar.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {% if page.author_profile or page.editor_profile or layout.author_profile or page.sidebar %}
   <div class="sidebar sticky">
   {% if page.author_profile or layout.author_profile %}{% include author-profile.html %}{% endif %}
-  {% if page.editor_profile or layout.editor_profile %}{% include editor-profile.html %}{% endif %}
+  {% if page.editor_profile or layout.editor_profile %}{% if page.author == page.editor %}{% else %}{% include editor-profile.html %}{% endif %}{% endif %}
```
Comment (berdandy) [inline docs/_layouts/build-sidebar.html]: same thing here with empty if-else

File: docs/_includes/build-sidebar.html [hunk 3]
```diff
@@ -1,9 +1,10 @@
 {% if page.author_profile or layout.author_profile or page.sidebar %}
   <div class="sidebar sticky">
   {% if page.author_profile or layout.author_profile %}{% include author-profile.html %}{% endif %}
-  {% if page.editor_profile or layout.editor_proflie %}
-    <hr/>
-    {% include editor-profile.html %}
+  {% if page.editor_profile or layout.editor_profile %}
```
Comment (berdandy) [inline docs/_includes/build-sidebar.html:4]: nice catch with proflie typo

File: docs/_posts/2022-04-19-power-bladesworn.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+## Variants
+
+Berserker can be replaced by Marauder and Knight, though knight will perform considerably worse.
+
+Keep your sigils the same on this one, as crit is very important and force is pretty strong.
+
+## Other References
+
+Check out [my video discussing showcasing](https://www.youtube.com/watch?v=AaPuiW0ulGM) this build.
```
Comment (berdandy) [inline docs/_posts/2022-04-19-power-bladesworn.md]: "discussing and showcasing"?

File: docs/_posts/2022-04-19-power-bladesworn.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
+---
+layout: build
+author: muwum
+editor: muwum
```
Comment (berdandy) [inline docs/_posts/2022-04-19-power-bladesworn.md]: Just leave off editor

**Review by berdandy — CHANGES_REQUESTED** [review]
I'd like a couple of minor things changed, when you get a chance.

Note that the front matter `editor` is entirely optional, so you could have just left out that field. However, I think I'll keep your editor-hiding commit.

File: docs/_posts/2022-04-19-power-bladesworn.md (hunk 4 above)
Comment (MuwuM) [reply docs/_posts/2022-04-19-power-bladesworn.md]: "discussing" was a usual copy & paste :/

File: docs/_includes/build-sidebar.html (hunk 1 above)
Comment (MuwuM) [reply docs/_includes/build-sidebar.html]: done ... wasn't too sure about the syntax during testing. thanks for the guidiance

File: docs/_layouts/build-sidebar.html (hunk 2 above)
Comment (MuwuM) [reply docs/_layouts/build-sidebar.html]: done

File: docs/_posts/2022-04-19-power-bladesworn.md (hunk 5 above)
Comment (MuwuM) [reply docs/_posts/2022-04-19-power-bladesworn.md]: done

**Review by berdandy — APPROVED** [review]
Looks good

**Pull request merged**